geode-core/src/test/java/org/apache/geode/distributed/internal/membership/MembershipDependenciesJUnitTest.java

```diff
@@ -100,6 +99,9 @@
       .onlyDependOnClassesThat(
           resideInAPackage("org.apache.geode.distributed.internal.membership.gms..")
               .or(resideInAPackage("org.apache.geode.internal.serialization.."))
+              .or(type(LogService.class))
```

Comment:
Maybe in a separate PR we can create a new package name for the classes in `geode-logging` then change these `type` exceptions to `resideInAPackage` exceptions.  That way we can be sure that `LogService`, `LoggingExecutors`, and `LoggingThread` do not reside in `geode-core`.

Reply:
I'd like to see all of these logging classes eventually end up packed under `org.apache.geode.logging` so these internal classes would end up under `org.apache.geode.logging.internal`. Because of how universally `LogService` is used, this will touch many more classes. So I'm ok with leaving further re-packaging to a later PR.

Reply:
Wilco.